operate/client/src/modules/utils/processInstanceDetailsDiagram.ts

```diff
@@ -11,33 +11,55 @@ import type {
   BusinessObjects,
 } from 'bpmn-js/lib/NavigatedViewer';
 
-const hasMultipleScopes = (
-  parentFlowNode?: BusinessObject,
-  totalRunningInstancesByFlowNode?: Record<string, number>,
+const checkScope = (
+  parentFlowNode: BusinessObject | undefined,
+  totalRunningInstancesByFlowNode: Record<string, number> | undefined,
+  predicate: (count: number) => boolean,
 ): boolean => {
-  if (parentFlowNode === undefined) {
+  if (!parentFlowNode) {
     return false;
   }
 
-  const totalRunningInstancesCount =
-    totalRunningInstancesByFlowNode?.[parentFlowNode.id];
-
-  const scopeCount = totalRunningInstancesCount ?? 0;
+  const count = totalRunningInstancesByFlowNode?.[parentFlowNode.id] ?? 0;
 
-  if (scopeCount > 1) {
+  // If predicate matches, return immediately
+  if (predicate(count)) {
     return true;
   }
 
+  // If not inside a SubProcess, stop recursion
   if (parentFlowNode.$parent?.$type !== 'bpmn:SubProcess') {
     return false;
   }
 
-  return hasMultipleScopes(
+  // Recurse upwards
+  return checkScope(
```

Comment:
Comments should explain why and not what but it's better to not have any comments at all since they might get out of sync with future changes.

I think we can simply remove them